Resources/public/js/views/fields/ez-relation-editview.js

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-relation-editview', function (Y) {
+    "use strict";
+    /**
+     * Provides the field edit view for the relation fields
+     *
+     * @module ez-relation-editview
+     */
+    Y.namespace('eZ');
+
+    var FIELDTYPE_IDENTIFIER = 'ezobjectrelation';
+
+    /**
+     * Relation edit view
+     *
+     * @namespace eZ
+     * @class RelationEditView
+     * @constructor
+     * @extends eZ.FieldEditView
+     */
+    Y.eZ.RelationEditView = Y.Base.create('relationEditView', Y.eZ.FieldEditView, [Y.eZ.AsynchronousView], {
+
+        events: {
+            '.ez-relationeditview-remove': {
+                'tap': '_removeRelation',
+            },
+            '.ez-discover': {
+                'tap': '_runUniversalDiscovery',
+            },
+        },
+
+        initializer: function () {
+            var that = this;
+
+            this._fireMethod = this._fireLoadFieldRelatedContent;
+            this._watchAttribute = 'destinationContent';
+            if( this.get('field').fieldValue.destinationContentId ){
+                this._set('destinationContentId', this.get('field').fieldValue.destinationContentId);
+            }
+            this.after("destinationContentChange", function () {
+                if ( that.get('destinationContent') ) {
```

Comment:
you don't need to use `that` here, this function is executed in the correct context